onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -274,24 +274,23 @@ def aten_angle(self: TensorType) -> TensorType:
 def aten_any(self: TTensor, dim: Optional[int] = None, keepdim: bool = True) -> BOOL:
     """any(Tensor self) -> Tensor"""
 
-    minus_1 = op.Constant(value_ints=[-1])
+    negative_one = op.Constant(value_ints=[-1])
     self_rank = op.Size(op.Shape(self))
     if self_rank == 0:
-        self = op.Reshape(self, minus_1)
+        self = op.Reshape(self, negative_one)
 
-    zero = op.Constant(value_float=0.0)
-    result = op.Not(op.Equal(self, zero))
+    result = op.Cast(self, to=BOOL.dtype)
```

Comment:
Possible to use a name that is not result for intermediate values?

Reply:
sure